### PR TITLE
TRA-3415 ignore unready pods

### PR DESCRIPTION
**Pull request by RamiBerm**

### CODE REVIEW

File: cli/kubernetes/provider.go [hunk 1]
```diff
@@ -635,3 +635,7 @@ func loadKubernetesConfiguration(kubeConfigPath string) clientcmd.ClientConfig {
 		},
 	)
 }
+
+func IsPodRunning(pod *core.Pod) bool {
```
Comment (IgorGov) [inline cli/kubernetes/provider.go:639]: why starts with capital letter? is it exposed method?

File: cli/cmd/tapRunner.go [hunk 2]
```diff
@@ -257,19 +253,15 @@ func watchPodsForTapping(ctx context.Context, kubernetesProvider *kubernetes.Pro
 		}
 	}
 	restartTappersDebouncer := debounce.NewDebouncer(updateTappersDelay, restartTappers)
-	timer := time.AfterFunc(time.Second*10, func() {
-		mizu.Log.Debugf("Waiting for URL...")
+
+	go func() {
```
Comment (IgorGov) [inline cli/cmd/tapRunner.go:257]: this one should be moved after the first call to "updateCurrentlyTappedPods"

File: cli/kubernetes/provider.go (hunk 1 above)
Comment (RamiBerm) [reply cli/kubernetes/provider.go:639]: fixed

File: cli/cmd/tapRunner.go (hunk 2 above)
Comment (RamiBerm) [reply cli/cmd/tapRunner.go]: fixed

**Review by IgorGov — APPROVED** [review]

**Pull request merged**